File: hw/medina/u1/hw07Dependences/newChickenJS/model/Poop.js
class Poop {
  /**
   * @param {number} amount - Amount of poop
   */
  constructor(amount) {
    this.amount = amount;
  }

  toString() {
    return "Poop{ amount = " + this.amount + " }";
  }

  getAmount() {
    return this.amount;
  }
}

module.exports = { Poop };

File: ws/gudino/u1/ws04Dependences/ChikenFarmSimulator/src/ec/edu/espe/chikenfarmsimulator/model/Poop.java
package ec.edu.espe.chikenfarmsimulator.model;

/**
 *
 * @author Bryan Gudino, KNOWLEDGE ENCAPSULATE, @ESPE
 */
public class Poop {
    private int amount;

    /**
     * @return the amount
     */
    public int getAmount() {
        return amount;
    }

    /**
     * @param amount the amount to set
     */
    public void setAmount(int amount) {
        this.amount = amount;
    }

    public Poop(int amount) {
        this.amount = amount;
    }

    @Override
    public String toString() {
        return "Poop{" + "amount=" + amount + '}';
    }
}
